liquidity-module/index.md

```diff
@@ -11,100 +11,113 @@ The liquidity module, known on the Cosmos Hub as Gravity DEX, enables users to c
 
 ## Use Liquidity Pools to Trade Tokens
 
-When using your Cosmos SDK-based blockchains, you want to enable users to trade tokens. You can have multiple tokens on your blockchain or have tokens from external blockchains sent to your blocking using inter-blockchain communication protocol (IBC). 
+When using Cosmos SDK-based blockchains, you want to enable users to trade tokens. You can have multiple tokens on your blockchain or have tokens from external blockchains sent to your blockchain using the inter-blockchain communication protocol (IBC). 
 
-The liquidity module allows users to use pools to trade those tokens on your blockchain. Each pool represents a token pair and allows the user to swap from one token to the other token.
+The liquidity module uses liquidity pools to trade tokens on your blockchain. Each pool represents a token pair and allows the user to swap assets from one token to another token.
 
 In this tutorial, you create your own blockchain, send tokens to another blockchain, create a pool, deposit to a pool, withdraw from a pool, and swap tokens.
 
-You can follow the [code with us session](https://www.youtube.com/watch?v=GxaqpzMk0jk&t=978s) for hands-on experience
+You can follow the [code with us session](https://www.youtube.com/watch?v=GxaqpzMk0jk&t=978s) for hands-on experience.
 
-**Important** In the code examples throughout this tutorial, when you see username be sure to substitute your username. 
+**Important:** In the code examples throughout this tutorial, when you see username be sure to substitute your username. 
 
 **You will learn how to:**
 
 - Create a blockchain with Starport
 - Create your own token on your blockchain
 - Connect your blockchain to the testnet
 - Send your own token with IBC to the testnet
-- Create a pool with your token
-- Use the pool with your token
-
+- Create a liquidity pool with your token
+- Use the liquidity pool with your token
 
 ## Requirements
 
 Before you start the tutorial, install the prerequisite software. 
 
 - [Install Starport](../starport/index.md) v0.16.2
```

Comment:
@fadeev hmmm looks like this tutorial needs updating for v0.17.0, so you know of any reason why we shouldn't do that change?

Reply:
Or should we just say latest binary. Because every time we update starport this will change..

Reply:
@barriebyron we have updated the relayer, so we may need to check if everything's still working.

@alijnmerchant21 that's the whole purpose of versioning 🙂 so that devs can use an old version that is guaranteed to work until we check that this also works with a newer version.